updatecli/updatecli.d/nodejs.yaml

```
@@ -0,0 +1,47 @@
+---
+name: Bump `nodejs` version according to packer-images
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  lastPackerImageVersion:
```

```suggestion
  lastPackerImageNodeVersion:
```

nit: the name is confusing: is it the last packer image version or nodeJS version?

---

That's an oversight 😅

Will fix it 🙂